### PR TITLE
Rzr-1: config environment

### PR DESCRIPTION
**Pull request by SashaGraves**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (cashidze) [inline .gitignore:1]: most probably we would need to add .vscode

File: package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "description": "UI part of the site of our friend",
+  "main": "app.js",
+  "scripts": {
+    "start": "parcel C:/Git/rzrsite-ui/index.html"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cashidze/rzrsite-ui.git"
+  },
+  "author": "",
```
Comment (cashidze) [inline package.json]: You probably would want to add your name here :)

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    // "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
+    // "strictNullChecks": true,              /* Enable strict null checks. */
+    // "strictFunctionTypes": true,           /* Enable strict checking of function types. */
+    // "strictBindCallApply": true,           /* Enable strict 'bind', 'call', and 'apply' methods on functions. */
+    // "strictPropertyInitialization": true,  /* Enable strict checking of property initialization in classes. */
+    // "noImplicitThis": true,                /* Raise error on 'this' expressions with an implied 'any' type. */
+    // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
+
+    /* Additional Checks */
+    // "noUnusedLocals": true,                /* Report errors on unused locals. */
```
Comment (cashidze) [inline tsconfig.json]: Uncomment

File: tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    // "noUnusedParameters": true,            /* Report errors on unused parameters. */
```
Comment (cashidze) [inline tsconfig.json]: Uncomment

File: tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    // "noImplicitReturns": true,             /* Report error when not all code paths in function return a value. */
+    // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
```
Comment (cashidze) [inline tsconfig.json]: Uncomment

File: index.tsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
+/* eslint react/destructuring-assignment: "off" */
```
Comment (cashidze) [inline index.tsx:1]: We shouldn't turn it off in real tsx files :)

Comment (SashaGraves) [reply index.tsx:1]: Ok, it's not)

**Review by cashidze — APPROVED** [review]

**Pull request merged**